transifex-migrate-resource.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python
+'''
+Run the script providing the slug of the project and the slug of the old and new resource.
+The new resource should already have been created.
+ 
+Example:
+Old resource slug: 'old'
+New resource slug: 'new'
+ 
+python transifex-migrate-resource.py project old new
+ 
+After running the command you will be asked for your Transifex username and password.
+'''
+# W.J. 2015 (license: MIT)
+# Based on https://gist.github.com/mapapage/bb07703afaa4ea590109 by Marilena Papageorgiou 
+
+from hashlib import md5
+import requests
+from requests.exceptions import RequestException
+import json
+import getpass
+import sys
+
+
+SERVER = "https://www.transifex.com"
+
+
+def get_source_entity_hash(context, key):
+    if isinstance(context, list):
+        if context:
+            keys = [key] + context
+        else:
+            keys = [key, '']
+    else:
+        if context:
+            keys = [key, context]
+        else:
+            keys = [key, '']
+    return str(md5(':'.join(keys).encode('utf-8')).hexdigest())
+
+
+def migrate_resource(project, from_slug, to_slug, AUTH):
+    headers = {'Content-type': 'application/json'}
+
+    langs_url = "/api/2/project/%s/languages/" % project
+    lang_entries = requests.get(
+        SERVER + langs_url,
+        headers=headers,
+        auth=AUTH
+    ).content
+    lang_entries = json.loads(lang_entries)
+
+    #lang_entries = [l for l in lang_entries if l['language_code']=='bg']
+
+    skip = True
+    for lang_entry in lang_entries:
+        if lang_entry['language_code']=='zh_CN.GB2312':
```

Comment:
Could you comment why this language is special cased?

Reply:
I don't know.